packages/expo-device/ios/EXDevice/EXDevice.m

```diff
@@ -207,30 +207,30 @@ + (NSNumber *)deviceYear
   
   // TODO: Apple TV and Apple watch
   NSDictionary *mapping = @{
-    // iPhone 1
-    @"iPhone1,1": @2007,
+    // // iPhone 1
+    // @"iPhone1,1": @2007,
     
-    // iPhone 3G
-    @"iPhone1,2": @2008,
+    // // iPhone 3G
+    // @"iPhone1,2": @2008,
     
-    // iPhone 3GS
-    @"iPhone2,1": @2009,
+    // // iPhone 3GS
+    // @"iPhone2,1": @2009,
     
-    // iPhone 4
-    @"iPhone3,1": @2010,
-    @"iPhone3,2": @2010,
-    @"iPhone3,3": @2010,
+    // // iPhone 4
+    // @"iPhone3,1": @2010,
+    // @"iPhone3,2": @2010,
+    // @"iPhone3,3": @2010,
     
-    // iPhone 4S
-    @"iPhone4,1": @2011,
+    // // iPhone 4S
+    // @"iPhone4,1": @2011,
     
-    // iPhone 5
-    @"iPhone5,1": @2012,
-    @"iPhone5,2": @2012,
+    // // iPhone 5
+    // @"iPhone5,1": @2012,
+    // @"iPhone5,2": @2012,
     
-    // iPhone 5S and 5C
-    @"iPhone5,3": @2013,
-    @"iPhone5,4": @2013,
+    // // iPhone 5S and 5C
+    // @"iPhone5,3": @2013,
+    // @"iPhone5,4": @2013,
```

Comment:
Remove these unless they serve a meaningful purpose. Also delete the corresponding entries from ModelNames.ts

Reply:
Maybe add a comment regarding all removed devices like: `all older devices are no longer supported and thus do not appear in this list anymore`? 🤔